types/three/examples/jsm/postprocessing/RenderPass.d.ts

```diff
@@ -1,13 +1,28 @@
-import { Scene, Camera, Material, Color } from '../../../src/Three';
-
+import {
+    Scene,
+    Camera,
+    Material,
+    Color,
+    WebGLMultipleRenderTargets,
+    WebGLRenderTarget,
+    WebGLRenderer,
+} from '../../../src/Three';
 import { Pass, FullScreenQuad } from './Pass';
 
 export class RenderPass extends Pass {
-    constructor(scene: Scene, camera: Camera, overrideMaterial?: Material, clearColor?: Color, clearAlpha?: number);
-    scene: Scene;
-    camera: Camera;
-    overrideMaterial: Material;
-    clearColor: Color;
+    constructor(scene?: Scene, camera?: Camera, overrideMaterial?: Material, clearColor?: Color, clearAlpha?: number);
+    scene?: Scene;
+    camera?: Camera;
+    overrideMaterial?: Material;
+    clearColor?: Color;
     clearAlpha: number;
     clearDepth: boolean;
+
+    render(
+        renderer: WebGLRenderer,
+        _: WebGLMultipleRenderTargets | WebGLRenderTarget | null,
+        writeBuffer?: WebGLMultipleRenderTargets | WebGLRenderTarget,
```

Comment:
Shouldn't the parameters be `readBuffer` and `writeBuffer` (in that order)?

Reply:
In three.js [RenderPass.js](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/postprocessing/RenderPass.js#L57), the first parameter is ignored and the second parameter is used as writeBuffer. 
The autocomplete always creates confusion since the params are not `write, read` but `ignored, write`

Reply:
So the variables are misnamed in the three.js source code?